libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -113,6 +142,18 @@ type instance MapError 'ConvAccessDenied = 'StaticError 403 "access-denied" "Con
 
 type instance MapError 'MLSNonEmptyMemberList = 'StaticError 400 "non-empty-member-list" "Attempting to add group members outside MLS"
 
+type instance MapError 'DuplicateMLSPublicKey = 'StaticError 400 "mls-duplicate-public-key" "MLS public key for the given signature scheme already exists"
+
+type instance MapError 'KeyPackageRefNotFound = 'StaticError 404 "mls-key-package-ref-not-found" "A referenced key packages could not be mapped to a known client"
+
+type instance MapError 'UnsupportedMLSMessage = 'StaticError 422 "mls-unsupported-message" "Attempted to send a message with an unsupported combination of content type instance MapError ' and wire format"
```

Comment:
The single quote within the message seems like something is off. Have you forgotten to finish the message?